From: Xuhui Lin <xuhui.lin@rock-chips.com>
Subject: tools: rkcommon: Add RV1103B support
Date: Tue, 07 Jul 2026 09:31:02 -0600

Allow rkcommon to generate an image for the RV1103B.

Signed-off-by: Xuhui Lin <xuhui.lin@rock-chips.com>
Signed-off-by: Fabio Estevam <festevam@nabladev.com>
Signed-off-by: Simon Glass <sjg@chromium.org>
---

(no changes since v1)

 tools/rkcommon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/rkcommon.c b/tools/rkcommon.c
index b39777fc060..6f25a1cda31 100644
--- a/tools/rkcommon.c
+++ b/tools/rkcommon.c
@@ -153,6 +153,7 @@ static struct spl_info spl_infos[] = {
 	{ "rk3568", "RK35", 0x10000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rk3576", "RK35", 0x80000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rk3588", "RK35", 0x100000 - 0x1000, false, RK_HEADER_V2 },
+	{ "rv1103b", "110E", 0x40000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rv1108", "RK11", 0x1800, false, RK_HEADER_V1 },
 	{ "rv1126", "110B", 0x10000 - 0x1000, false, RK_HEADER_V1 },
 };
-- 
2.43.0

